src/vars/styles/global.css

```diff
@@ -5,16 +5,15 @@ body {
   transition: 0.2s;
   margin: 0;
   padding: 0;
-  -webkit-app-region: drag;
```

Comment:
Afaik you cant drag the app around if you remove this.

Reply:
You should be able to drag the app around by the top bar area.